en/docs/security-announcements/security-advisories/2021/WSO2-2021-1699.md

```diff
@@ -60,8 +57,12 @@ In addition, this update removes the vulnerable "JndiLookup'' class from the fol
 * org.wso2.ei.analytics.elk
 * siddhi-store-elasticsearch
 
+### IMPACT
+By leveraging the vulnerability, a malicious actor may perform Remote Code Execution by providing a crafted payload.
 
-Note that WSO2 is aware, and has analyzed the CVE-2021-45105, and the associated update of Log4j (2.17.0). The default product configurations of WSO2 products are not vulnerable to CVE-2021-45105. However, WSO2 strongly recommends executing the below command from the WSO2 product-home, and confirming that the customized configurations you use are not vulnerable.
+### SOLUTION
+
+We highly recommend migrating to the latest version of the respective WSO2 products to permanently mitigate the identified vulnerabilities. Alternatively, you can use the script provided below to temporarily address the issue in your current deployment. Please note that this temporary fix is fully effective in resolving the identified vulnerability.
```

Comment:
"script provided below" is to check if someone has exploited the deployment. We need to bting the mitigation script to this advisory or point to the incident-clarification from here. I prefer pointing to incident clarification.